src/draw/lv_draw_buf.c

```diff
@@ -147,6 +147,8 @@ lv_draw_buf_t * lv_draw_buf_create(uint32_t w, uint32_t h, lv_color_format_t cf,
     void * buf = lv_draw_buf_malloc(size, cf);
     LV_ASSERT_MALLOC(buf);
     if(buf == NULL) {
+        LV_LOG_ERROR("No memory: %"LV_PRIu32"x%"LV_PRIu32", cf: %d, stride: %"LV_PRIu32", %"LV_PRIu32"Byte, ",
```

Comment:
It's not necessarily a warning. LVGL can try malloc-ing next time. 
Let's make is a warning or info.

Reply:
I have changed to warning for now. Let's get back to it when standalone image cache is added, which will give us the ability to try mallocing on failed.